public/css/cons.css

```diff
@@ -0,0 +1,63 @@
+nav{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap;
+}
+.nav--right{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    margin:5px 20px;
+    align-items: center;
+    width: 40%;
+    font-family: 'Aref Ruqaa', sans-serif;
+    font-size: 24px;
+}
+.nav--right img{
+    margin-left:50px;
+    width: 80px;
+}
+.nav__main{
+    text-decoration: none;
+    color: #A3BBF8;
+    justify-content: center;
+    margin-left:22px;
+    margin-bottom: 2px;
+
+}
+.nav__main:hover, .nav__prev:hover, .nav__logout:hover{
+    border-bottom: 3px solid #A3BBF8;
+}
```

Comment:
@naremanhilles You could reduce some of the duplicated styles with .nav__main, .nav__prev and .nav__logout using another selector similar to the one on line 30:

```
.nav__main, .nav__prev, .nav__logout {
  text-decoration: none;
  justify-content: center;
  margin-bottom: 2px;
}
```

Another alternative is to add a common "nav__item" class to all of those elements (in addition to the class they already have, like nav__main or nav__logout).  Then all of the styles could go into this common "nav__item" class, and another benefit is that you could add additional nav items in the future and they would look correct once you gave them the nav__item class, without you having to add anything to your css file.